tests/TODO.md

```diff
@@ -0,0 +1,43 @@
+### DESCRIPTION
+
+This file contains a list of test cases and notes on features which should
+be covered with automatic tests.
+
```

Comment:
This description has to be extended with:
* this file's structure - how it is organized
* schema of test case - HOWTO for adding new test cases and description for understanding existent test cases' structure